example/ios/Flutter/.last_build_id

```diff
@@ -0,0 +1 @@
+bdb137e74343ba89ec4409af66025a5f
```

Comment:
`.last_build_id` shouldn't be committed. 
Please add to our gitignore `**/ios/Flutter/.last_build_id` to prevent this.

Reply:
Good point! Done in https://github.com/PSPDFKit/pspdfkit-flutter/pull/68/commits/25976b2e4ac9201d9816a79ad9e6229a5e30f74b